1-exercises/I-string-replace/exercise.js

```diff
@@ -13,7 +13,13 @@
 let story =
   "I like dogs. One day I went to the park and I saw 10 dogs. It was a great day.";
 
-let result = story.replace("", "");
+let result = story
+  .replace("dogs", "cats")
+  .replace("day", "night")
+  .replace("10", "100000")
+  .replace("dogs", "cats")
+  .replace("great", "brilliant")
+  .replace("day", "night");
 
```

Comment:
Whilst this is technically correct, it's a bit fragile. If the story included another dog, this code would fail. Initially, I thought that the replace method would replaces all occurrences of the first argument, but I learned that's not correct. Here's my version:
```javascript
let result = story
  .replace(/day/g, "night")
  .replace("great", "brilliant")
  .replace(/dogs/g, "cats")
  .replace("10", "100000");
```
The first argument to replace can be a regular expression between two "/" characters. the trailing "g" means match the expression throughout the input string, so any number of "day" and "dogs" occurrences will be correctly replaced.